src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/services/GetPersonService.kt

```diff
@@ -13,6 +13,14 @@ class GetPersonService(
   @Autowired val prisonerOffenderSearchGateway: PrisonerOffenderSearchGateway,
   @Autowired val probationOffenderSearchGateway: ProbationOffenderSearchGateway
 ) {
+
+  fun execute(firstName: String, lastName: String): List<Person?> {
```

Comment:
I don't think that this should be here, instead it should be self-contained in its own service called `GetPersonsService` that has this method description, see https://github.com/madetech/clean-architecture/blob/master/use_case.md#use-case (we call them services but same thing).

>In code, the entry point of a Use Case is a class that has one public method.

This should also help remove the need to add extra describes in tests.

Reply:
Fair enough, the justification was that we're using the same endpoint so we had grouped them together. But let's stick to the MT way, that makes more sense to me :)

Reply:
After making this change, the benefits to the tests alone are very nice and I'm fully on board!